lib/gherkin/gherkin.ex

```diff
@@ -72,7 +72,7 @@ defmodule Gherkin do
       outline = %Gherkin.Elements.ScenarioOutline{}
       Gherkin.scenarios_for(outline) |> Enum.each(&run_scenario/1)
   """
-  def scenarios_for(outline = %Elements.ScenarioOutline{name: name, tags: tags, steps: steps, examples: examples}) do
+  def scenarios_for(_outline = %Elements.ScenarioOutline{name: name, tags: tags, steps: steps, examples: examples}) do
```

Comment:
If we aren't using it (perhaps at one time between commits it was being used 😆 ), then we can just remove the variable as the struct is descriptive enough, dont you think?